src/net/client/mod.rs

```diff
@@ -0,0 +1,213 @@
+use super::Command;
```

Comment:
unfortunately git has decided this is a brand new file... i'll try to point to the new pieces of code